ROADMAP.md

```diff
@@ -9,11 +9,11 @@ We target a new release of ASO approximately every two months, though this may v
 Our current release plan:
 
 | Version | Estimated Release |
-| ------- | ----------------- |
-| [2.6.0](https://github.com/Azure/azure-service-operator/issues?q=is%3Aopen+is%3Aissue+milestone%3Av2.6.0) | Mid February 2024 |
+| ------- | -----------------:|
 | [2.7.0](https://github.com/Azure/azure-service-operator/issues?q=is%3Aopen+is%3Aissue+milestone%3Av2.7.0) | Mid April 2024 |
 | 2.8.0 | Mid June 2024 |
```

Comment:
There are links for 2.8.0 at least now as well (maybe 2.9.0 too?)

Reply:
Added.